experiment-train/skill.yaml

```diff
@@ -0,0 +1,26 @@
+camel: 1.0.0
+name: experiment-train
+title: Experiment Train
+description: Experiment Train
+inputs:
+  - name: params
+    title: Job Input
+    parameters:
+      - name: params
+        type: object
+        description: Input Params
+        required: true
+    routing:
+      all:
+        action  : experiment-train
+        runtime: cortex/jobs
+        output: response
+
```

Comment:
output name of action and skill should be same else the skill will not get saved.